querybook/server/lib/query_analysis/validation/validators/presto_optimizing_validator.py

```diff
@@ -171,53 +214,117 @@ def get_query_validation_results(
                 like_strings = []
             token_idx += 1
 
-        return validation_errors
+        return validation_results
 
 
-class PrestoOptimizingValidator(BaseQueryValidator):
-    def languages(self):
-        return ["presto", "trino"]
+class ColumnNameSuggester(BasePrestoSQLGlotDecorator):
```

Comment:
I guess column name suggester is not specific to presto, other engines, liek sparksql can also benefit from it?

Reply:
Refactored so that we can use it with other query engines